doc/sphinx/src/using.rst

```diff
@@ -128,6 +128,65 @@ Please note that none of these functions are thread or MPI aware. In a parallel
 
 as appropriate.
 
+robust_utils.hpp
+^^^^^^^^^^^^^^^^^^^
+
+``robust_utils.hpp`` contains small utility functions for numerical
+robustness, especially around floating point numbers. The available
+functionality is contained in the namespace ``PortsOfCall::Robust`` and includes:
+
+* ``constexpr auto SMALL<T>()`` returns a small number of type ``T``.
+* ``constexpr auto EPS<T>()`` returns a value of type ``T`` close to machine epsilon.
+* ``constexpr auto min_exp_arg<T>()`` returns the smallest of type ``T`` safe value to pass into an exponent.
+* ``constexpr auto max_exp_exp_arg<T>()`` returns the max value of type ``T`` to pass into an exponent.
+* ``auto make_positive(const T val)`` makes the argument of type ``T`` positive.
+
+where here all functionality is templated on type ``T`` and marked
+with ``PORTABLE_INLINE_FUNCTION``. The default type ``T`` is always
+``Real``.
+
+The function
+
+.. code-block:: cpp
+
+  template <typename T>
+  PORTABLE_FORCEINLINE_FUNCTION
+  Real make_bounded(const T val, const T vmin, const T vmax);
+
+bounds ``val`` between ``vmin`` and ``vmax``.
```

Comment:
It's probably worth noting that it's exclusive of the bounds themselves

Reply:
Totally optional here, but it might also be interesting to note the similarities and differences to [std::clamp](https://en.cppreference.com/w/cpp/algorithm/clamp).

Reply:
Yep will do.